sefaria/views.py

```diff
@@ -1364,20 +1364,60 @@ def modtools_upload_workflowy(request):
     if request.method != "POST":
         return jsonResponse({"error": "Unsupported Method: {}".format(request.method)})
 
-    file = request.FILES['wf_file']
+    # Handle both single and multiple file uploads
+    files = []
+    if 'wf_file' in request.FILES:
+        # Single file upload (backward compatibility)
+        files = [request.FILES['wf_file']]
+    elif 'workflowys[]' in request.FILES:
+        # Multiple file upload
+        files = request.FILES.getlist("workflowys[]")
+    else:
+        return jsonResponse({"error": "No files provided. Use 'wf_file' for single file or 'workflowys[]' for multiple files."})
+
+    # Handle checkbox parameters - support both boolean and string formats
     c_index = request.POST.get("c_index", False)
+    if isinstance(c_index, str):
+        c_index = c_index.lower() == 'true'
```

Comment:
why not

```suggestion
c_index = request.GET.get('c_index', '').lower() == 'true'
```
with no condition?
also, maybe we should have a util function for getting bool query param (and it seems that in most views we use 0/1 for booleans).